packages/sociaml-facebook-api/sociaml-facebook-api.0.4.1/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "1.2"
+name: "sociaml-facebook-api"
+version: "0.4.1"
+maintainer: "dominic.price@nottingham.ac.uk"
+homepage: "https://github.com/dominicjprice/sociaml-facebook-api"
+authors: [ "Dominic Price" ]
+license: "ISC"
+available: [ ocaml-version >= "4.02.1" ]
+bug-reports: "https://github.com/dominicjprice/sociaml-facebook-api/issues"
+dev-repo: "https://github.com/dominicjprice/sociaml-facebook-api"
```

Comment:
I believe this should end in `.git` but I don't see any errors or lint failures from that. @AltGr, has this restriction been lifted?